Modix.Data/Repositories/InfractionRepository.cs

```diff
@@ -199,23 +202,54 @@ public async Task<IReadOnlyCollection<InfractionSummary>> SearchSummariesAsync(I
                 .ToArrayAsync();
 
         /// <inheritdoc />
-        public async Task<IDictionary<InfractionType, int>> GetInfractionCountsAsync(InfractionSearchCriteria searchCriteria)
+        public async Task<IReadOnlyCollection<InfractionCount>> GetInfractionCountsAsync(ulong guildId, ulong subjectId)
```

Comment:
Why are we reducing functionality here?

Reply:
I mean, I *could* try to fit in all the possible scenarios that come with the search criteria, but so far we haven't needed the extra functionality for this query, and I don't foresee this query being used in a wide variety of cases.